packages/core/helper-plugin/lib/src/components/DateTimePicker/DateTimePicker.stories.mdx

```diff
@@ -0,0 +1,60 @@
+<!--- DateTimePicker.stories.mdx --->
+
+import { useState } from 'react';
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import DateTimePicker from './index';
+
+<Meta title="DateTimePicker" component={DateTimePicker} />
```

Comment:
would it be possible to move the story to components?
`<Meta title="components/DateTimePicker" component={DateTimePicker} />`